packages/common.vm/tools/vm.common/vm.common.psm1

```diff
@@ -266,11 +260,13 @@ function VM-Install-Shortcut{
         [string] $toolName,
         [Parameter(Mandatory=$true, Position=1)]
         [string] $category,
-        [Parameter(Mandatory=$true, Position=2)]
+        [Parameter(Mandatory=$false, Position=2)]
         [string] $executablePath,
         [Parameter(Mandatory=$false)]
         [bool] $consoleApp=$false,
         [Parameter(Mandatory=$false)]
+        [bool] $powershell=$false,
```

Comment:
[nitpick] it could be a switch instead of a bool. Or is there a reason for that?

Reply:
I agree. It probably could do better as a switch. :)